src/libs/scroll-to-top-on-collapse.js

```diff
@@ -0,0 +1,22 @@
+export default () => {
+	console.log('hihi')
+	const btns = document.querySelectorAll('.file-actions > .js-details-target')
+	const prToolbar = document.querySelector('.pr-toolbar')
+	const toolbarHeight = prToolbar.getBoundingClientRect().height
```

Comment:
`prToolbar` might not exist on the PR's Conversation tab, you'll need a condition or just move it inside the event handler.